src/nvim/tui/input.c

```diff
@@ -705,6 +705,7 @@ static void handle_raw_buffer(TermInput *input, bool force)
         const size_t bufsize = termkey_get_buffer_size(input->tk);
         const bool success = termkey_set_buffer_size(input->tk, MAX(bufsize + delta, bufsize * 2));
         assert(success);
+        (void)success;
```

Comment:
Better to use abort https://github.com/neovim/neovim/pull/26309#discussion_r1410902728

Reply:
I don't think it should abort in Release builds.

Reply:
Why? If we fail to allocate, we always abort.